docs/src/terminology.md

```diff
@@ -6,7 +6,13 @@ The following terms are used throughout the documentation.
 
 ## account
 
-A persistent file addressed by [public key](terminology.md#public-key) and with [lamports](terminology.md#lamport) tracking its lifetime.
+A record in the Solana ledger that either holds data or is an executable program.
+
+Like an account at a traditional bank, a Solana account may hold funds called [lamports](terminology.md#lamport). Like a file in Linux, it is addressable by a [key](terminology.md#public-key) (public key or pubkey).
```

Comment:
> it is addressable by a [key](terminology.md#public-key) (public key or pubkey).

This part is not quite right.  An account address may be one of:
1. an ed25519 public key
2. a program-derived account address (32byte value forced off the ed25519 curve)
3. a hash of an ed25519 public key with a 32 character string